### PR TITLE
Add a `get_coeffs` method 

### PR DESCRIPTION
**Pull request by mmpagani**
This PR adds a method through which is possible to get the values for all the periods/frequencies for a set of coefficient names. This is useful for https://github.com/gem/oq-engine/issues/7433

### CODE REVIEW

**Comment by micheles** [comment]
What about the property
```python
    @property
    def non_sa_coeffs(self):
        return {imt: self._coeffs[imt] for imt in self._coeffs
                if imt.string[:2] != 'SA'}
```
used in
```diff
openquake/hazardlib/gsim/nga_east.py
293:    coeffs.update(phi_model.non_sa_coeffs)
316:    coeffs.update(phi_model.non_sa_coeffs)
840:    phi_std.update(self.PHI_SS.non_sa_coeffs)
847:            PHI_S2SS_MODEL[self.phi_s2ss_model].non_sa_coeffs)

openquake/hazardlib/gsim/projects/acme_base.py
32:    coeffs.update(phi_model.non_sa_coeffs)

openquake/hazardlib/tests/gsim/kotha_2020_test.py
```
What happens if there are EAS coefficients? Is this a case that may happen?

**Comment by mmpagani** [comment]
I saw the `non_sa_coeffs` property but I do not think it matches what I need. The method I am adding returns two matrixes that I can easily use to do a vectorised calculation for a set of sites. `non_sa_coeffs` instead returns a dictionary. However, it's true that this property must also be updated to support EAS.

**Comment by mmpagani** [comment]
More in general we should add ad-hoc tests for the `CoeffsTable` class.

File: openquake/hazardlib/gsim/coeffs_table.py [hunk 1]
```diff
@@ -182,6 +184,30 @@ def non_sa_coeffs(self):
         return {imt: self._coeffs[imt] for imt in self._coeffs
                 if imt.string[:2] != 'SA'}
 
+    def get_coeffs(self, coeff_list):
```
Comment (micheles) [inline openquake/hazardlib/gsim/coeffs_table.py:187]: I would expect a ``get_coeffs(self, kind, coeff_names)`` where kind can be "SA" or "EA" and then you can use ``imt.startswith(kind)``

**Comment by micheles** [comment]
We can change `get_coeffs` later, so for the moment LGTM as it is.

**Pull request merged**